docs/source/index.rst

```diff
@@ -4,6 +4,15 @@ Hyperledger Iroha documentation
 
 .. image:: ../image_assets/iroha_logo.png
 
+.. warning::
+    Please note that Hyperledger Iroha v1 is **no longer under active development** and the support for Iroha v1 is limited. The core development team focuses on `Hyperledger Iroha v2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_, a complete rewrite of Iroha in Rust.
+    
+    For new projects, **we recommend using Iroha v2 instead of Iroha v1**. If you are using Iroha v1, we encourage you to plan an upgrade and switch to using Iroha v2.
+    
+    The long-term supported version of Iroha v2 (`iroha2-lts`) is your best choice if you want to use Iroha in production and get tech support from our side. Unfortunately, we cannot offer the same support for Iroha v1, so please consider using Iroha v2.
```

Comment:
```suggestion
    The long-term supported version of Iroha v2 (`iroha2-lts`) is your best choice if you want to use Iroha in production and receive tech support from our side. Unfortunately, we cannot offer the same support for Iroha v1, so please consider using Iroha v2.
```

Reply:
Alternatively, we can remove the "so please consider using..." statement altogether